tests/vary.mjs

```diff
@@ -80,6 +80,72 @@ export default {
         }
       ]
     },
+    {
+      name: 'An optimal HTTP cache stores and correctly serves multiple object variants when the `Vary` response header value depends on one or more request header values',
+      id: 'vary-dynamic',
+      kind: 'optional',
+      browser_skip: true,
```

Comment:
In theory this could apply to browsers too; it's just that they don't currently store more than one variant.

You could filter for that by making it `depends_on: ['vary-invalidate']`